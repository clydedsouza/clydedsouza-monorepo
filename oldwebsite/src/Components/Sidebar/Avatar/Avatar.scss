@import '../../../Sass/variables';

.avatar {
  width: 180px;
  height: 180px;
  min-height: 180px;
  border-radius: 120px;
  background-image: url('./clyde2024.jpg');
  background-position: center;
  background-size: cover;
  border: 6px solid $contrast-accent;
  margin: 2vh 0;

  @include for-tablets-down {
    width: 120px;
    height: 120px;
    min-height: 120px;
    margin: 0px auto !important;
    border: 4px solid $contrast-accent;
  }

  @include for-large-desktop-up {
    width: 210px;
    height: 210px;
    min-height: 210px;
  }
}
