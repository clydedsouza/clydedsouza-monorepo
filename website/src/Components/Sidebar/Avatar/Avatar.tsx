import './Avatar.scss'

function Avatar() {
  return <div className="avatar"></div>
}

export default Avatar
