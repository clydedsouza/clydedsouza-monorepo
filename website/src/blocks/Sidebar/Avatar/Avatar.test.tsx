import { render } from "@testing-library/react";
import Avatar from "./Avatar";

describe("Avatar", () => {
  it("should render avatar", () => {
    const { container } = render(<Avatar />);
    expect(container).toMatchSnapshot();
  });
});
