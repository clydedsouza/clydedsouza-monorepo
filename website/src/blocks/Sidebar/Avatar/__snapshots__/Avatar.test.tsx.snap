// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render avatar 1`] = `
<div>
  <div
    class="avatar"
  />
</div>
`;
